app/views/passengers/index.haml

```diff
@@ -49,7 +49,7 @@
           - unless @filters.include? 'Active'
             %td.yes= yes_no_image passenger.active?
           %td.yes= yes_no_image passenger.permanent?
-          %td.yes= passenger.expiration_display
+          %td.yes{data: {sort: sortable_date(passenger.doctors_note)}}=passenger.expiration_display
```

Comment:
On the one-hand, I asked myself why this column has a class that _appears_ to be only for yes-no icons.

On the other hand, that class turns out to not really have anything to do with yes-no icons, it just makes the text centered :roll_eyes: (As does the identical, unused, class `.no`)

On the third hand, why are expiration dates centered when no other text is?

Reply:
> why are expiration dates centered when no other text is?

because it looks goddamn weird.
![Screen Shot 2019-09-26 at 11 16 33 AM](https://user-images.githubusercontent.com/9645316/65701341-24720080-e04f-11e9-8fdc-2f651b16fb9f.png)

Like okay, I can rename/remove some classes but I'm not 100% sure I want to get down deep into why the headers and columns don't line up well in this case.

Reply:
Oh it looks weird because the headers are also centered. I did *not* see that.

Reply:
I got to delete some stuff!